infoschema/tables.go

```diff
@@ -1368,6 +1371,14 @@ var tableDeadlocksCols = []columnInfo{
 	{name: "TRX_HOLDING_LOCK", tp: mysql.TypeLonglong, size: 21, flag: mysql.NotNullFlag, comment: "The transaction ID (start ts) of the transaction that's currently holding the lock"},
 }
 
+var tableDataLockWaitsCols = []columnInfo{
+	{name: "HASH", tp: mysql.TypeLonglong, size: 21, flag: mysql.NotNullFlag | mysql.UnsignedFlag, comment: "The hash of the lock in TiKV's LockManager "},
```

Comment:
Is it useful since we have the key?

Reply:
IMO it is useful because wait manager works on hash instead of key.

Reply:
We can change the waiter manager to use key.